.github/workflows/test.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
       contents: read
       pull-requests: write
     steps:
+      - name: print env
```

Comment:
cab this be removed ?

Reply:
@rpolansky suggested to actually leave it for debugging, no secrets are there